tests/README.md

```diff
@@ -110,28 +98,12 @@ These tests are located in a subfolder (`test`) within a feature's directory and
 | $FEATURE_NAME | test_$TEST_NAME.py | features/$FEATURE_NAME/test/test_$TEST_NAME.py |
 | CIS | test_cis.py | [features/cis/test/test_cis.py](../features/cis/test/test_cis.py) |
 
-### Running Unit Tests
-Basic unit tests are executed after each build by the `chroot` platform. However, it may be necessary to run tests in a running system instead. Therefore, the `kvm` platform can be used.
-
-#### Automated Unit Tests
-As described, `chroot` tests are automatically performed after each build. By default, this doesn't include the `kvm` platform and may be added by appending `--tests	chroot,kvm` to the `build.sh`.
+### Running Unit tests
 
-Example: `./built.sh kvm-dev --tests chroot,kvm`
-
-#### chroot
-`chroot` tests may also be executed on a given artifact afterwards. A full documentation regarding running the unit tests on a `chroot` platform can be found [here](https://github.com/gardenlinux/gardenlinux/tree/main/tests#chroot).
-
-#### KVM
-`kvm` tests may also be executed on a given artifact afterwards. A full documentation regarding running the unit tests on a `kvm` platform can be found [here](https://github.com/gardenlinux/gardenlinux/tree/main/tests#kvm).
+To run unit tests for a specific target, use the command `./test ${target}.`
 
 
 # Integration Tests
```

Comment:
Do we really want to remove this section? Even though the ~~Integration~~Platform are automated nowadays, we still should have a description how to run them manually.

Reply:
You are right, we should keep a description how the ~integration~Platform tests can be executed manually. 

The chapter "Integration Tests" keeps the description now (was removed previously before my last change)
The chapter "Unit tests" removes the reference, and only describes the `./tests` method